README.md

```diff
@@ -99,6 +99,209 @@ And here is another example on Linux:
+ 0x7ffc26411000: [MemBlock_t(BaseAddress=0x7ffc26411000, AllocationBase=0x7ffc26410000, AllocationProtect=0x80, RegionSize=0x11c000)],
+ 0x7ffc2652d000: [MemBlock_t(BaseAddress=0x7ffc2652d000, AllocationBase=0x7ffc26410000, AllocationProtect=0x80, RegionSize=0x49000)],
+ 0x7ffc26576000: [MemBlock_t(BaseAddress=0x7ffc26576000, AllocationBase=0x7ffc26410000, AllocationProtect=0x80, RegionSize=0x1000)],
+ 0x7ffc26577000: [MemBlock_t(BaseAddress=0x7ffc26577000, AllocationBase=0x7ffc26410000, AllocationProtect=0x80, RegionSize=0x2000)],
+ 0x7ffc26579000: [MemBlock_t(BaseAddress=0x7ffc26579000, AllocationBase=0x7ffc26410000, AllocationProtect=0x80, RegionSize=0x9000)],
+ 0x7ffc26582000: [MemBlock_t(BaseAddress=0x7ffc26582000, AllocationBase=0x7ffc26410000, AllocationProtect=0x80, RegionSize=0x86000)],
+ 0x7ffc26608000: [MemBlock_t(BaseAddress=0x7ffc26608000, AllocationBase=0x0, AllocationProtect=0x0, RegionSize=0x3d99e8000)]}
+```
+
+To facilitate the parsing in a human-friendly manor, some helper functions were provided:
+ * `udmp_parser.utils.TypeToString`: convert the region type to its meaning (from MSDN)
+ * `udmp_parser.utils.StateToString`: convert the region state to its meaning (from MSDN)
+ * `udmp_parser.utils.ProtectionToString`: convert the region protection to its meaning (from MSDN)
+
+This allows to search filter in a more comprehensible way:
+
+
+```python
+# Collect only executable memory regions
+>>> exec_regions = [region for _, region in dmp.Memory().items() if "PAGE_EXECUTE_READ" in udmp_parser.utils.ProtectionToString(region.Protect)]
+
+# Pick any, disassemble code using capstone
+>>> exec_region = exec_regions[-1]
+>>> mem = dmp.ReadMemory(exec_region.BaseAddress, 0x100)
+>>> for insn in cs.disasm(bytearray(mem), exec_region.BaseAddress):
+  print(f"{insn=}")
+
+insn=<CsInsn 0x7ffc26582000 [cc]: int3 >
+insn=<CsInsn 0x7ffc26582001 [cc]: int3 >
+insn=<CsInsn 0x7ffc26582002 [cc]: int3 >
+insn=<CsInsn 0x7ffc26582003 [cc]: int3 >
+insn=<CsInsn 0x7ffc26582004 [cc]: int3 >
+insn=<CsInsn 0x7ffc26582005 [cc]: int3 >
+insn=<CsInsn 0x7ffc26582006 [cc]: int3 >
+insn=<CsInsn 0x7ffc26582007 [cc]: int3 >
+insn=<CsInsn 0x7ffc26582008 [cc]: int3 >
+insn=<CsInsn 0x7ffc26582009 [cc]: int3 >
+insn=<CsInsn 0x7ffc2658200a [cc]: int3 >
+insn=<CsInsn 0x7ffc2658200b [cc]: int3 >
+insn=<CsInsn 0x7ffc2658200c [cc]: int3 >
+insn=<CsInsn 0x7ffc2658200d [cc]: int3 >
+insn=<CsInsn 0x7ffc2658200e [cc]: int3 >
+insn=<CsInsn 0x7ffc2658200f [cc]: int3 >
+insn=<CsInsn 0x7ffc26582010 [48895c2410]: mov qword ptr [rsp + 0x10], rbx>
+insn=<CsInsn 0x7ffc26582015 [4889742418]: mov qword ptr [rsp + 0x18], rsi>
+insn=<CsInsn 0x7ffc2658201a [57]: push rdi>
+insn=<CsInsn 0x7ffc2658201b [4156]: push r14>
+insn=<CsInsn 0x7ffc2658201d [4157]: push r15>
+[...]
+```
+
 # Authors
 
-* Axel '[@0vercl0k](https://twitter.com/0vercl0k)' Souchet
+* Axel [@0vercl0k](https://twitter.com/0vercl0k)' Souchet
```

Comment:
I think this change might be a mistake 😅.

What do you think to add a section:

```
# Contributors
- yourname: [Python bindings implementation](https://github.com/0vercl0k/udmp-parser/pull/6)
```

Reply:
What do you think of the change?

Reply:
Note - I will only appear when (if 😜?) my PR is accepted

Reply:
LGTM!